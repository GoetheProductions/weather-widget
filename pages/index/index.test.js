import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';
import Component from './index';
import mockData from '../../mockdata/copenhagen.json';

const component = renderer.create(<Component data={mockData} />);

test('Matches snapshot', () => {
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('Renders div as parent', () => {
  expect(component.toJSON().type).toEqual('div')
});
