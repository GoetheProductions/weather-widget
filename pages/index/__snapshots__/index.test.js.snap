// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches snapshot 1`] = `
<div>
  <div
    className="widget"
    style={
      Object {
        "margin": "10px",
        "width": "300px",
      }
    }
  >
    <div
      className="panel panel-info"
    >
      <div
        className="panel-heading"
      >
        Weather in 
        <b>
          Copenhagen
        </b>
      </div>
      <ul
        className="list-group"
      >
        <li
          className="list-group-item"
        >
          Temperature:
           
          <b>
            20.55Â°C
          </b>
        </li>
        <li
          className="list-group-item"
        >
          Humidity:
           
          <b>
            43
          </b>
        </li>
        <li
          className="list-group-item"
        >
          Wind:
           
          <b>
            4.6 m/s 
          </b>
        </li>
        <li
          className="list-group-item"
        >
          <form
            className="form-inline"
            onSubmit={[Function]}
          >
            <div
              className="form-group"
            >
              <input
                className="form-control"
                onChange={[Function]}
                placeholder="City"
                type="text"
                value=""
              />
            </div>
            <button
              className="btn btn-default"
              disabled={true}
              type="submit"
            >
              Search
            </button>
          </form>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
